incubator/hnc/pkg/controllers/setup.go

```diff
@@ -55,5 +55,14 @@ func Create(mgr ctrl.Manager, f *forest.Forest, maxReconciles int) error {
 		return fmt.Errorf("cannot create Hierarchy controller: %s", err.Error())
 	}
 
+	// Create the ConfigReconciler, passing it the object reconcillers so it can call them.
```

Comment:
Remove ", passing it the object reconcillers so it can call them" in the comment?

Reply:
Good catch! Removed.